releng/org.eclipse.xtext.sdk.target/org.eclipse.xtext.sdk.target.target

```diff
@@ -179,10 +179,14 @@
 		<unit id="org.eclipse.xtend" version="0.0.0"/>
 		<unit id="org.eclipse.xtend.typesystem.emf" version="0.0.0"/>
 		<unit id="org.eclipse.m2e.feature.feature.group" version="0.0.0"/>
-		<unit id="org.eclipse.buildship.feature.group" version="0.0.0"/>
 		<repository location="https://download.eclipse.org/releases/oxygen/201804111000"/>
 	</location>
 
+	<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="false" type="InstallableUnit">
+		<unit id="org.eclipse.buildship.core" version="0.0.0"/>
+		<repository location="https://download.eclipse.org/releases/2018-12"/>
```

Comment:
am not sure if we want to pull such and old version here.
in the diff job we install other stuff manually too

Reply:
Indeed this is the wrong direction. I'll discard this PR.